src/main/scala/no/ndla/searchapi/model/grep/GrepElement.scala

```diff
@@ -0,0 +1,10 @@
+/*
+ * Part of NDLA search-api.
+ * Copyright (C) 2020 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.searchapi.model.grep
+
+case class GrepElement(kode: String, tittel: Seq[GrepTitle])
```

Comment:
For kompetansemål treng nok også koblinger til kompetansemålsett og eventuellt andre ting. Mykje er nok felles men her må vi nok skille på typer som vi henter.

Reply:
Kikket både her https://data.udir.no/kl06/v201906/kompetansemaal-lk20 og her https://data.udir.no/kl06/v201906/kompetansemaalsett-lk20 men ser ikke noe kobling mellom de, hvordan skal vi koble dem sammen?

Reply:
Et kompetansemålsett har en liste med kompetansemål: https://data.udir.no/kl06/v201906/kompetansemaalsett-lk20/KV6
Og et kompetansemål har en tilhoerer-kompetansemaalsett:
https://data.udir.no/kl06/v201906/kompetansemaal-lk20/KM39